src/test/crimson/seastore/test_transaction_manager.cc

```diff
@@ -566,6 +566,43 @@ struct transaction_manager_test_t :
     bool success = try_submit_transaction(std::move(t));
     EXPECT_FALSE(success);
   }
+
```

Comment:
Seems like this would be easier to trigger in test_btree_lba_manager.cc since you wouldn't need to actually create the extents.

Reply:
the similar logic code works fine in test_btree_lba_manager.cc , can't trigger find_hole assert:  see below, 

  auto trigger_find_hole(const size_t& len, int &num) {
    return repeat_eagain([&, this] {
      return seastar::do_with(create_transaction(), [&, this] (auto &t) {
        return crimson::do_for_each(
               boost::make_counting_iterator(0),
               boost::make_counting_iterator(num),
          [&, this] (auto idx) {
          return with_trans_intr(*t.t, [&, this] (auto &transaction) {
            using ertr = LBAManager::alloc_extent_iertr;
            using ret = ertr::future<>;
            return lba_manager->alloc_extent(transaction, L_ADDR_MIN, len, get_paddr())
              .si_then([&, this] (auto &&pin) ->ret {
              EXPECT_EQ(len, pin->get_length());
              auto [b, e] = get_overlap(t, pin->get_laddr(), len);
              EXPECT_EQ(b, e);
              t.mappings.emplace(std::make_pair(pin->get_laddr(),
                                 test_extent_t{pin->get_paddr(), pin->get_length(), 1}
                                 ));
              return ertr::make_ready_future<>();
            });
          });
        }).safe_then([&t, this] {
          return submit_transaction(std::move(t.t));
        }).safe_then([&t, this]() {
          test_lba_mappings.swap(t.mappings);
        });
      });
    }).handle_error(
      crimson::ct_error::assert_all{
      "Invalid error in SeaStore::list_collections"
      }
    );
  }

Reply:
Hmm, I see.  I'll try this out once my changes to the lba btree are done (probably end of this week).